userDAO.js

```diff
@@ -0,0 +1,15 @@
+// development dependencies
+var app = require('express')();
+var DA0 = require('../DOAfactory');
+var db = require('../..database');
+
+db.connect();
+
+var UserDAO = DAO("UserDAO", client,
```

Comment:
Mixed double and single quotes.<br>'DAO' is not defined.<br>'client' is not defined.